os/kernel/pthread/pthread_keydelete.c

```diff
@@ -60,6 +60,7 @@
 #include <errno.h>
 #include <debug.h>
 
+#include "sched/sched.h"
```

Comment:
#if CONFIG_NPTHREAD_KEYS > 0 is needed.

Reply:
I modified not to build those files which are used in pthread key functionality so that we don't need to make a conditional inside.